docs/recipes/build/rollup.md

```diff
@@ -0,0 +1,135 @@
+# [Rollup](https://github.com/rollup/rollup) Bundling
+
+## Dependencies
+
+- A working installation of `node` (4.0+) & `npm`.
+- `npm install --save-dev sw-build sw-lib rollup rollup-plugin-node-resolve`
+
+## Service Worker Code
+
+This approach assumes that your `src/sw.js` file is using [ES2015 module
```

Comment:
This could be part of the intro - i.e. explanation of if you are using Rollup and would like to use ES2015 modules in your service worker - here is how you can get the manifest revision details.